web_app/routes/finder_routes.py

```diff
@@ -18,13 +19,17 @@ def finder():
     if not filtered_markets.empty:
         print("The following farmers markets are available in", user_city + ",", user_state, ":")
         for index, row in filtered_markets.iterrows():
-            market_info = {
-                "Market Name": row["market_name"],
-                "Market Addressddress": f"{row['market_street']}, {row['market_city']} +, {row['market_state']} {row['market_zip']}"
-                "-" * 20
-            }
-            market_list.append(market_info)
+            #market_info = {
+            #    "Market Name": row["market_name"],
+            #    "Market Address": f"{row['market_street']}, {row['market_city']} +, {row['market_state']} {row['market_zip']}"
+            #}
+            #market_list.append(market_info)
+            # keep dictionary original keys, we can combine more flexibly into a desired display format on the page
+            market_list.append(dict(row))
```

Comment:
instead of joining together all the market info, we can keep the different fields separate, which will allow us more flexibility / options when we display this info on the page